apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -235,6 +252,13 @@ export const BulkImportUserWizard: FunctionComponent<BulkImportUserInterface> =
         getUserStoreList();
     }, [ userstore ]);
 
+    /**
+     * Set the default multiple invites option.
+     */
+    useEffect(() => {
+        setConfigureMode(MultipleInviteMode.MANUAL);
```

Comment:
Without using this in an `useEffect` shall we directly assign as the initial value ?

```
    const [ configureMode, setConfigureMode ] = useState<string>(MultipleInviteMode.MANUAL);
```

Reply:
Improved with 9fd857accd3f35459c4b56b50a2d17744b271f36.